src/index.js

```diff
@@ -1,66 +1,42 @@
-const express = require('express'),
-  bodyParser = require('body-parser'),
-  session = require('express-session'),
-  cors = require('cors'),
-  errorhandler = require('errorhandler'),
-  mongoose = require('mongoose');
+import errorhandler from 'errorhandler';
+import express, { json, urlencoded } from 'express';
+import { config } from 'dotenv';
+import cors from 'cors';
+import Debug from 'debug';
+import routes from './routes';
 
-const isProduction = process.env.NODE_ENV === 'production';
+config();
+
+const debug = Debug('dev');
 
-// Create global app object
+const { PORT } = process.env;
 const app = express();
 
+app.use(json());
+app.use(urlencoded({ extended: false }));
 app.use(cors());
+app.use('/api/v1', routes);
 
-// Normal express config defaults
-app.use(require('morgan')('dev'));
 
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
+app.use('*', (request, response) => {
+  response.status(404).send('Not Found');
+});
 
-app.use(require('method-override')());
 
-app.use(express.static(`${__dirname}/public`));
+app.get('/', (request, response) => {
+  response.status(200).send('1kbIdeas');
+});
 
-app.use(
-  session({
-    secret: 'authorshaven',
-    cookie: { maxAge: 60000 },
-    resave: false,
-    saveUninitialized: false
-  })
-);
+const isProduction = process.env.NODE_ENV === 'production';
 
 if (!isProduction) {
```

Comment:
Kindly remove this code block and the errorhandler package since you are already handling the error on line 36.